components/Main/Card.js

```diff
@@ -53,9 +83,15 @@ export default function MainPage() {
         <ContainerStyle key={recipe._id}>
```

Comment:
Here is the point where you render a list of ``<div>``s. You should implement a ``<ul>`` with ``<li>``s here as well.

Reply:
This is not changed yet.

Reply:
You are rendering a LIST of recipies. So please make sure, that there is a ``<ul>`` that contains ``<li>``s that contain the ``recipe``.
This is for semantic html very important.